### PR TITLE
Translate to Portuguese the Solidity course - Section 1 - Lesson 2

### PR DESCRIPTION
**Pull request by igorln**

### CODE REVIEW

File: Solidity_And_Smart_Contracts/pt/Section_1/Lesson_2_Write_Your_First_Smart_Contract.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+pragma solidity ^0.8.0;
+```
+
+Esta é a versão do compilador Solidity que queremos que nosso contrato use. Ele basicamente diz "ao executar isso, eu só quero usar a versão 0.8.0 do compilador Solidity, nada inferior. Obs: certifique-se de que a versão do compilador é a mesma em `hardhat.config.js`.
+
+```solidity
+import "hardhat/console.sol";
+```
+
+Alguma magia que nos foi dada pelo Hardhat para fazer alguns console logs em nosso contrato. Na verdade, é desafiador debugar contratos inteligentes, mas esta é uma das vantagens que o Hardhat nos oferece para tornar isso um mamão com açúcar.
```
Comment (daniserafs) [inline Solidity_And_Smart_Contracts/pt/Section_1/Lesson_2_Write_Your_First_Smart_Contract.md:46]: I believe the word "alguma" is way more vague in Portuguese, do you think changing to "uma" makes more sense?

File: Solidity_And_Smart_Contracts/pt/Section_1/Lesson_2_Write_Your_First_Smart_Contract.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+```solidity
+contract WavePortal {
+    constructor() {
+        console.log("Yo yo, I am a contract and I am smart");
+    }
+}
+```
+
+Portanto, os contratos inteligentes parecem uma espécie de `class` em outras linguagens, se você já viu isso! Assim que inicializarmos este contrato pela primeira vez, o construtor executará e imprimirá essa linha. Por favor, faça essa linha dizer o que você quiser :)!
```
Comment (daniserafs) [inline Solidity_And_Smart_Contracts/pt/Section_1/Lesson_2_Write_Your_First_Smart_Contract.md:56]:
```suggestion
Portanto, se você já viu isso, os contratos inteligentes parecem uma espécie de `class` em outras linguagens! Assim que inicializarmos este contrato pela primeira vez, o construtor executará e imprimirá essa linha. Por favor, faça essa linha dizer o que você quiser :)!
```
I thinks it's a bit confusing to leave "se você já viu isso" at the end of de sentence.
```suggestion
Portanto, os contratos inteligentes parecem uma espécie de `class` em outras linguagens, se você já viu isso! Assim que inicializarmos este contrato pela primeira vez, o construtor executará e imprimirá essa linha. Por favor, faça essa linha dizer o que você quiser :)!
```

**Review by daniserafs — COMMENTED** [review]
Awesome work here! I left some suggestions, let me know what you think of them!

File: Solidity_And_Smart_Contracts/pt/Section_1/Lesson_2_Write_Your_First_Smart_Contract.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+👩‍💻 Vamos escrever um contrato
+----------------------------
+
+Incrível, nós conseguimos.
+
+Vamos pular direto para o nosso projeto.
+
+Vamos escrever um contrato inteligente que nos permita enviar um 👋 para o nosso contrato e manter o controle do número total de ondas. Isso será útil porque, em seu site, você pode querer acompanhar isso! Sinta-se à vontade para alterar isso para se adequar ao seu caso.
```
[the remark this answers is not in the document]
Comment (igorln) [reply Solidity_And_Smart_Contracts/pt/Section_1/Lesson_2_Write_Your_First_Smart_Contract.md:8]: Really doesn't make sense, I will fix it

**Comment by igorln** [comment]
@daniserafs fixed what you said, can you take another look?

**Review by daniserafs — APPROVED** [review]

**Comment by helloitsm3** [comment]
Closing as we do not have any way to verify the contents at the moment. Do re-open if you feel that this needs another review.

**Pull request closed without merging**